### PR TITLE
MCLOUD-7387: Add MFTF documentation for AWS S3

### PR DESCRIPTION
**Pull request by shiftedreality**
### Fixed Issues (if relevant)

1. https://jira.corp.magento.com/browse/MCLOUD-7387

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/verification tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)
 - [ ] Changes to Framework doesn't have backward incompatible changes for tests or have related Pull Request with fixes to tests

### CODE REVIEW

File: docs/configuration.md [hunk 1]
```diff
@@ -368,6 +368,66 @@ When pause is enabled, MFTF will generate pause() command in _failed() hook so t
 ENABLE_PAUSE=true
 ```
 
+### REMOTE_STORAGE_AWSS3_DRIVER
+
+The remote storage driver, to enable AWS S3 use `aws-s3`
```
Comment (dshevtsov) [inline docs/configuration.md:373]:
```suggestion
The remote storage driver. To enable AWS S3, use `aws-s3`.
```

File: docs/configuration.md [hunk 2]
```diff
@@ -368,6 +368,66 @@ When pause is enabled, MFTF will generate pause() command in _failed() hook so t
...
+
+Example:
+
+```conf
+REMOTE_STORAGE_AWSS3_ACCESS_KEY=MY_ACCESS_KEY
+```
+
+### REMOTE_STORAGE_AWSS3_SECRET_KEY
+
+The optional secret ket to access S3 bucket.
```
Comment (dshevtsov) [inline docs/configuration.md:423]:
```suggestion
The optional secret key to access S3 bucket.
```

**Review by dshevtsov — CHANGES_REQUESTED** [review]

**Comment by jilu1** [comment]
@magento import pr to magento-commerce/magento2-functional-testing-framework

**Comment by magento-engcom-team** [comment]
@jilu1 the pull request successfully imported.

**Comment by jilu1** [comment]
@magento import pr to magento-commerce/magento2-functional-testing-framework

**Comment by magento-engcom-team** [comment]
@jilu1, an error occurred during the Pull Request import.

**Comment by jilu1** [comment]
@magento import code to magento-commerce/magento2-functional-testing-framework

**Comment by magento-engcom-team** [comment]
@jilu1, an error occurred during the Pull Request import.

**Comment by jilu1** [comment]
Manually created internal pr https://github.com/magento-commerce/magento2-functional-testing-framework/pull/15 with updated content

**Pull request merged**